scripts/fmt.sh

```diff
@@ -16,6 +16,11 @@
 pre-commit run --all-files
 RETURN_CODE=$?
 
+## cat this file for helping on identifying the root cause when some issue happens
+if [ -f /$USER/.cache/pre-commit/pre-commit.log ]; then
+  cat /$USER/.cache/pre-commit/pre-commit.log
```

Comment:
Good idea to print the actual errors. However, I would prefer to keep the Makefile working consistently on Mac and Linux, but as the  `/$USER/` dir won't likely exist on Mac, the pre-commit errors will not show up there.

I have `pre-commit` installed on my local system and I run `make fmt` on all the modelmesh projects, without having to spin up a Docker container.

On a resource restricted machine it makes sense to not require unnecessary overhead like having to spin up a Docker engine.

I think all the messages can be printed to the console with the `-verbose` flag. If we want to only see verbose output on error, we could `cat` the log file as you did, but we should not assume a file location.

If we want to use the pre-commit log file, then we should configure it like described here:
https://pre-commit.com/#config-log_file

i.e. 

<table><tr><td>
  <img width="513" alt="log_file: .pre-commit.log" src="https://github.com/kserve/rest-proxy/assets/12246093/3bdaee72-9175-4201-8015-5d93f8770f8c">
</td></tr></table>

Reply:
updated, please take a look.